pom.xml

```diff
@@ -1423,34 +1452,12 @@
             </goals>
             <configuration>
               <includeScope>test</includeScope>
-              <outputFile>${test_classpath_file}</outputFile>
+	      <outputProperty>test_classpath</outputProperty>
             </configuration>
           </execution>
         </executions>
       </plugin>
 
-      <!-- This plugin reads a file into maven property. And it lets us write groovy !! -->
```

Comment:
@ScrapCodes you added this in https://github.com/apache/spark/commit/daaca14c16dc2c1abc98f15ab8c6f7c14761b627 -- looks OK to you?